tests/test_models.py

```diff
@@ -277,7 +282,7 @@ def test_frontend_plot(self):
     def test_frontend_PoolingWindows(self):
         im = plt.imread(op.join(DATA_DIR, 'nuts.pgm'))
         im = torch.tensor(im, dtype=DTYPE, device=DEVICE).unsqueeze(0).unsqueeze(0)
-        frontend = po.simul.Front_End()
+        frontend = po.simul.FrontEnd(pretrained=True, requires_grad=False)
```

Comment:
Can we add some tests with `pretrained=False` and `requires_grad=True` (both together and separately)? Just to make sure all the expected functionality works (forward, synthesis, plotting) and that the filters change as they're being learned

Reply:
Added True/False combos for ``pretrained`` and ``requires_grad`` for all relevant tests, except for PoolingWindows FrontEnd tests where only ``requires_grad`` can be True/False; bc if the weights are untrained, the responses are nonsensical. 

This is bc right now, weights can be anything, despite the fact that the model should be a series of diff of Gaussians. This will be addressed in #88.